openlibrary/plugins/openlibrary/js/Share.js

```diff
@@ -0,0 +1,17 @@
+const popup = document.querySelector('.popup'),
+    field = popup.querySelector('.copy-field'),
+    input = field.querySelector('input'),
+    copy = field.querySelector('button');
+copy.onclick = () => {
+    input.select(); //select input value
+    if (document.execCommand('copy')) {
+        //if the selected text copy
+        field.classList.add('active');
+        copy.innerText = 'Copied';
+        setTimeout(() => {
+            window.getSelection().removeAllRanges(); //remove selection from document
+            field.classList.remove('active');
+            copy.innerText = 'Copy';
+        }, 3000);
+    }
+};
```

Comment:
```suggestion
export function initShareModal($modalLinks){
    $modalLinks.each(function() {
        this.addEventListener('click', function() {
            $.colorbox({
                inline: true,
                opacity: '0.5',
                href: `#social-modal-content`,
                width: '100%',
                maxWidth: '400px'
            });
        })
    })
}
```
Let's initialize the click listener here.  This way we avoid adding unneeded data to the `SocialShare.html` template.